configure.ac

```diff
@@ -49,7 +49,7 @@ PKG_CHECK_MODULES(INITIAL_SETUP,
                   pango >= $PANGO_REQUIRED_VERSION
                   rest-0.7
                   json-glib-1.0
-		  libsecret-1
+                  libsecret-1
```

Comment:
I put the commits to submit upstream in this branch https://github.com/kinvolk/gnome-initial-setup/tree/alessandro/upstream-patches-3-22.
@cosimoc Can you submit them?